app/services/web_mentions/web_mention_handler.rb

```diff
@@ -0,0 +1,44 @@
+module WebMentions
+  class WebMentionHandler
+    def initialize(canonical_url, article_url = nil, webmention_url = nil)
+      @canonical_url = canonical_url
+      @article_url = article_url
+      @webmention_url = webmention_url
+    end
+
+    def self.call(*args)
+      new(*args).call
+    end
+
+    def call
+      accepts_webmention? ? send_webmention : Rails.logger.info("#{@canonical_url} doesn't support Webmentions")
+    end
+
+    def accepts_webmention?
+      webmention_url.blank? ? false : true
+    end
+
+    def webmention_url
+      document = Nokogiri::HTML.parse(URI.open(@canonical_url)).at_css("link[rel='webmention']")
```

Comment:
Sorry, I pressed enter too early in my previous review. Can we use `HTTParty.get` instead of `URI.open`? That's our standard interface to the external world, mostly